apps/indexer/lib/indexer/block/fetcher.ex

```diff
@@ -84,7 +84,7 @@ defmodule Indexer.Block.Fetcher do
   # These are all the *default* values for options.
   # DO NOT use them directly in the code.  Get options from `state`.
 
-  @receipts_batch_size 250
+  @receipts_batch_size 100
```

Comment:
@carterqw2 @rkachowski how do we feel about it?

Reply:
Looks like smaller batch sizes is a way to go in general.

Reply:
agree with both of you, I haven't noticed that receipts are a bottleneck, although reducing a single batch size is (usually) a good idea, worth trying out if this affects us anyhow and also knowing why it's being reduced here